acceptance-tests/src/test/scala/ch/renku/acceptancetests/workflows/Collaboration.scala

```diff
@@ -45,38 +50,14 @@ trait Collaboration {
     click on projectPage.Collaboration.Issues.tab sleep (1 second)
   }
 
-  def `verify there are no issues`: Unit = {
-    Then("they should see a 'No issues to display' info")
-    verify userCanSee projectPage.Collaboration.Issues.noIssues
+  def `verify that the GitLab issues iFrame is visible`: Unit = {
+    Then("they should see the GitLab issues page in an iFrame")
+    verify userCanSee projectPage.Collaboration.Issues.collaborationIframe
   }
 
-  def `create a new issue`(title: String, description: String): Unit = {
-    When("the user navigates to the Collaboration tab")
-    click on projectPage.Collaboration.tab sleep (1 second)
-    And("they navigate to the Issues sub tab")
-    click on projectPage.Collaboration.Issues.tab sleep (2 seconds)
-    And("the user clicks on the 'New Issue' button")
-    click on projectPage.Collaboration.Issues.newIssueLink sleep (1 second)
-
-    And("they fill out the form")
-    `create an issue with title and description`(title, description)
-  }
-
-  def `view the issue`(issueTitle: String): Unit =
-    `try few times before giving up` { _ =>
-      Then("the new issue should be displayed in the list")
-      val issueTitles = projectPage.Collaboration.Issues.issueTitles
-      if (issueTitles.size < 1) fail("There should be at least one issue")
-      issueTitles.find(_ == issueTitle) getOrElse fail("Issue with expected title could not be found.")
-    }
-
-  def `create an issue with title and description`(title: String, description: String): Unit = {
-    val tf = projectPage.Collaboration.Issues.NewIssue.titleField
-    tf.clear() sleep (1 second)
-    tf enterValue title
-    click on projectPage.Collaboration.Issues.NewIssue.markdownSwitch sleep (1 second)
-    projectPage.Collaboration.Issues.NewIssue.descriptionField enterValue description
-    click on projectPage.Collaboration.Issues.NewIssue.createIssueButton sleep (1 second)
+  def `verify that the GitLab issues page link is visible`: Unit = {
+    Then("they should see a link to the GitLab issues page")
+    verify userCanSee projectPage.Collaboration.Issues.gitLabIssuesLink
   }
 
   def `verify branch was added`: Unit = {
```

Comment:
I see you left this here... will we show the banner for merge requests inside renku? otherwise we can maybe delete this code.

Reply:
You are right! I have removed this method since the banner is gone.